zerver/lib/test_runner.py

```diff
@@ -569,7 +570,8 @@ def get_test_names(suite: unittest.TestSuite) -> List[str]:
         suite = cast(TestSuite, suite)
         return [full_test_name(t) for t in get_tests_from_suite(suite)]
 
-def get_tests_from_suite(suite: TestSuite) -> TestCase:
+# Type the suite as `Any` as mypy can't see unittest.suite.TestSuite.
+def get_tests_from_suite(suite: Any) -> TestCase:
```

Comment:
This comment doesn’t seem accurate; `unittest.suite.TestSuite` works fine here.

Reply:
Yep, not sure what I was doing here before to have caused that.